### PR TITLE
Automation Savings Plan docs

### PR DESCRIPTION
**Pull request by TarikFD**
Added a title, assembly, and modules for the automation savings planner docs.

(6/9) Doc preview here: http://file.rdu.redhat.com/kevchin/analytics/automation-savings-planner/build/tmp/en-US/html-single/ (VPN required)

### CODE REVIEW

File: downstream/assemblies/analytics/assembly-automation-savings-planner.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-automation-savings-planner"]
+
+:context: assembly-automation-savings-planner-ctxt
+
+The automation savings planner allows you to scope the size of your automation work by creating automation savings plans. You can create an automation savings plan by outlining a list of steps needed to fully automate your work. Each savings plan also provides tangible metrics, such as estimated job duration or completed tasks, for you to better understand and compare your automation jobs.
+
+The following procedures introduces the ways to get started with your automation savings plans by creating new plans or managing existing plans for your automation needs.
```
Comment (cbudz) [inline downstream/assemblies/analytics/assembly-automation-savings-planner.adoc:9]: introduce w/o the 's' at the end here for agreement.

File: downstream/modules/analytics/proc-create-savings-plan.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Create an automation savings plan by defining the tasks needed to complete an automation job using the {planner}.
+
+. Navigate to *Red Hat Insights* > *Savings Planner*.
+. Click *Add Plan*.
+. Provide information about your automation job using the provided form:
+.. Enter descriptive information, such as a name, description, and type of automation.
+.. Enter technical information, such as the number of hosts, the duration to manually complete this job, and how often you complete this job.
+.. Click *Next*.
+
+[NOTE]
```
Comment (cbudz) [inline downstream/modules/analytics/proc-create-savings-plan.adoc:18]: This note may be better placed by taking the elements mentioned and moving them up to introductory sentence. Maybe as bullet points for visibility?

File: downstream/modules/analytics/con-review-savings-calculations.adoc [hunk 3]
```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+// assembly-automation-savings-planner.adoc
+
+
+[id="con-review-savings-calculations_{context}"]
+
+= Review savings calculations for your automation plans
```
Comment (cbudz) [inline downstream/modules/analytics/con-review-savings-calculations.adoc:7]: Are there any RBAC requirements to access these features?

**Review by cbudz — CHANGES_REQUESTED** [review]
Hey @TarikFD just a few comments here.

File: downstream/modules/analytics/con-review-savings-calculations.adoc (hunk 3 above)
Comment (TarikFD) [reply downstream/modules/analytics/con-review-savings-calculations.adoc:7]: RBAC will be implemented at a later date, so not at the moment. When it does I'll have to come back to these and add prerequisites based on role permissions

**Comment by TarikFD** [comment]
Made some changes @cbudz . Please see the preview link for the updated build and lmk what you think

**Comment by cbudz** [comment]
Hey @TarikFD changes look good. Merging this one

**Review by cbudz — APPROVED** [review]

**Pull request merged**